### DISCUSSION TITLE
How can I remove outline from collapse when focused

### DISCUSSION THREAD

**Discussion by BilalAtique** [saadeghi/daisyui#2343] [Q&A] [answered]
I want to remove the outline from collapse when focused. I have tried:
`<div tabIndex={0} className="collapse-arrow collapse focus-visible:outline-none">` 
But it is not working
![image](https://github.com/saadeghi/daisyui/assets/74999164/97a153d2-7152-4597-8358-1529d1696d83)

**Comment by saadeghi** [comment, marked as answer]
It works. Here's an example:
https://play.tailwindcss.com/ehhpS3vC89
